trikset/ti-linux-kernel
# ov7670+VPIF: 640x480, 320x240, 160x120 @ 30fps

how to check:
* make modules, not only zImage
* insmod ov7670.ko
* use this script for a low fps check in any of the resolutions:
```
# params: width height fps major-device-no
# works with width heigh fps equal to:
# 320 240 8
# 640 480 2
GST_DEBUG=3 gst-launch-1.0 \
  v4l2src device=/dev/video$4 ! "video/x-raw,format=NV16,width=$1,height=$2,framerate=30/1" ! \
  videorate ! "video/x-raw,framerate=$3/1" ! \
  videoscale ! 'video/x-raw,width=320,height=240' ! \
  videoconvert ! 'video/x-raw,format=RGB' !  \
  videoflip method=clockwise !   videoconvert ! ximagesink 
```
* use this one for 160x120 only with 30 fps 
```
# params: 160 120 30 major-device-no
GST_DEBUG=3 gst-launch-1.0 \
  v4l2src device=/dev/video$4 ! "video/x-raw,format=NV16,width=$1,height=$2,framerate=30/1" ! \
  videorate ! "video/x-raw,framerate=$3/1" ! videoconvert !  ximagesink 
```
